tests/integration/models/device.spec.js

```diff
@@ -661,6 +695,40 @@ describe('Device Model', function () {
 			);
 		});
 
+		describe('balena.models.device.getMetrics()', function () {
+			givenAnApplication(before);
+			givenADevice(before, {
+				cpu_usage: 34,
+				memory_usage: 1000, // 1GB in MiB
```

Comment:
Is 1 MiB 1000 or 1024? See https://en.wikipedia.org/wiki/Mebibyte, it seems it is 1024?

Reply:
@CameronDiver @richbayliss can you clarify what this comment should be saying?
Maybe `memory_usage: 1024, // 1GB in MiB`?